client/js/components/button/BackToTopButton.vue

```diff
@@ -52,7 +52,7 @@ export default {
       this.scrollPos = document.documentElement.scrollTop
       const fromBottom = (this.contentHeight - 10 - this.footerheight > this.scrollPos) ? 10 : -this.contentHeight + 10 + this.footerheight + this.scrollPos
 
-      this.buttonPosition = `bottom: ${fromBottom}px; left: ${this.positionFromLeft}px; position: fixed`
+      this.buttonPosition = `bottom: ${fromBottom}px; left: ${this.positionFromLeft}px; position: sticky; z-index: 1`
```

Comment:
when I test this, the Positioning is a bit off and the button sits on the content and not aligned with the menu anymore. Is it the same when you test it? OR is it myabe some issue with my "on the fly" tweak in the browser?
![Screenshot_2023-07-21_12-03-09-back-to-top](https://github.com/demos-europe/demosplan-core/assets/40487461/8ef82562-01e4-4f3a-8907-e88ae4e3bfce)